packages/eas-cli/src/build/build.ts

```diff
@@ -206,15 +207,19 @@ export function handleBuildRequestError(error: any, platform: Platform): never {
       `You have already reached the maximum number of pending ${requestedPlatformDisplayNames[platform]} builds for your account. Try again later.`
     );
   } else if (error?.graphQLErrors) {
-    const graphQLError = error.graphQLErrors[0];
-    const requestIdLine = graphQLError?.extensions?.requestId
-      ? `\nRequest ID: ${graphQLError.extensions.requestId}`
-      : '';
-    const originalErrorMessage = graphQLError?.message
-      ? `\nError message: ${graphQLError.message}`
-      : '';
+    const errorMessage = error.graphQLErrors
+      .map((graphQLError: GraphQLError) => {
```

Comment:
Are we sure that items on this list will always be of the `GraphQLError` type?

Previously they were treated as `any` type and access to their fields was conditional. After your changes, it is no longer conditional and I wonder if this is 100% safe.

Reply:
From `urql` site:
```The graphQLErrors property may be an array that contains [normalized GraphQLErrors as they were received in the errors array from a GraphQL API.](https://graphql.org/graphql-js/error/)```
Hence I believe they are always GraphQLErrors